### PR TITLE
doc/dev/release-process.rst: New container build/release process

### PR DESCRIPTION
**Pull request by dmick**
## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Review by zdover23 — APPROVED** [review]

**Comment by dmick** [comment]
This documents the differences from https://github.com/ceph/ceph/pull/60755

**Comment by zdover23** [comment]
Looks good from a grammar perspective. Someone with more technical knowledge will have to verify the non-grammar part of this.

File: doc/dev/release-process.rst [hunk 1]
```diff
@@ -136,7 +136,9 @@ See `the Ceph Tracker wiki page that explains how to write the release notes <ht
 
 #. Obtain the sha1 of the version commit from the `build job <https://jenkins.ceph.com/view/all/job/ceph>`_ or the ``sha1`` file created by the `ceph-setup <https://jenkins.ceph.com/job/ceph-setup/>`_ job.
 
-#. Download the packages from chacra.ceph.com to the signing virtual machine. These packages get downloaded to ``/opt/repos`` where the `Sepia Lab Long Running (Ceph) Cluster <https://wiki.sepia.ceph.com/doku.php?id=services:longrunningcluster>`_ is mounted.
+#. Download the packages from chacra.ceph.com to the signing virtual machine. These packages get downloaded to ``/opt/repos`` where the `Sepia Lab Long Running (Ceph) Cluster <https://wiki.sepia.ceph.com/doku.php?id=services:longrunningcluster>`_ is mounted.  Note: this step will also run a command to download the
```
Comment (anthonyeleven) [inline doc/dev/release-process.rst:139]: Suggest “command to transfer”

**Review by anthonyeleven — APPROVED** [review]

File: doc/dev/release-process.rst (hunk 1 above)
Comment (dmick) [reply doc/dev/release-process.rst:139]: fair.  changed and repushed.

**Comment by zdover23** [comment]
@dmick, Should this be backported to Squid, Reef, and Quincy?

**Comment by dmick** [comment]
@zdover23 yes, it should, as should the code it documents (still working on that backport at the moment)

**Review by yuriw — APPROVED** [review]

**Comment by dmick** [comment]
I think this is sufficiently reviewed and ready for merge.  @zdover23 do you want me to do anything to facilitate the backports?

**Comment by zdover23** [comment]
> I think this is sufficiently reviewed and ready for merge. @zdover23 do you want me to do anything to facilitate the backports?

I'll take care of them.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/60971 - Squid backport
https://github.com/ceph/ceph/pull/60972 - Reef backport
https://github.com/ceph/ceph/pull/60973 - Quincy backport

**Comment by dmick** [comment]
Thanks @zdover23

**Pull request merged**